pkg/nack/generator_interceptor.go

```diff
@@ -147,14 +151,43 @@ func (n *GeneratorInterceptor) loop(rtcpWriter interceptor.RTCPWriter) {
 
 				for ssrc, receiveLog := range n.receiveLogs {
 					missing := receiveLog.missingSeqNumbers(n.skipLastN)
+
+					if len(missing) == 0 || n.nackCountLogs[ssrc] == nil {
+						n.nackCountLogs[ssrc] = map[uint16]uint16{}
```

Comment:
Is the new map a concern on every loop? Majority case is going to be `len(missing) == 0`. So, re-creating a map for all ssrcs every 100ms. Is that okay?

Reply:
My understanding is that an empty map is actually nil internally (i.e. no allocation), so I think this is OK.